dispatches/workflow/train_market_surrogates/dynamic/do_train_surrogates.py

```diff
@@ -0,0 +1,78 @@
+#################################################################################
+# DISPATCHES was produced under the DOE Design Integration and Synthesis
+# Platform to Advance Tightly Coupled Hybrid Energy Systems program (DISPATCHES),
+# and is copyright (c) 2021 by the software owners: The Regents of the University
+# of California, through Lawrence Berkeley National Laboratory, National
+# Technology & Engineering Solutions of Sandia, LLC, Alliance for Sustainable
+# Energy, LLC, Battelle Energy Alliance, LLC, University of Notre Dame du Lac, et
+# al. All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. Both files are also available online at the URL:
+# "https://github.com/gmlc-dispatches/dispatches".
+#################################################################################
+
+import os
+from Simulation_Data import SimulationData
+from Train_NN_Surrogates import TrainNNSurrogates
+from Time_Series_Clustering import TimeSeriesClustering

```

Comment:
These import paths should be absolute (starting with `dispatches.workflow.train_market_surrogates`) or relative (starting with one or more `.`). For more information see e.g. https://realpython.com/absolute-vs-relative-python-imports/